.github/workflows/release.yaml

```
@@ -66,6 +66,12 @@ jobs:
       - name: Archive the OpenAPI JSON schemas
         run: |
           tar -czvf ./output/crd-schemas.tar.gz -C schemas .
+      - name: Setup Syft
+        run: |
+          curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | \
+          sh -s -- -b /usr/local/bin ${SYFT_VERSION}
+        env:
+          SYFT_VERSION: "0.35.1"
```

This either needs to be 

```suggestion
          sh -s -- -b /usr/local/bin v${SYFT_VERSION}
        env:
          SYFT_VERSION: "0.35.1"
```

or:


```suggestion
          sh -s -- -b /usr/local/bin ${SYFT_VERSION}
        env:
          SYFT_VERSION: "v0.35.1"
```

---

Good catch, I've added the `v` prefix.